FlatCache/FlatCache.swift

```diff
@@ -9,17 +9,8 @@
 import Foundation
 
 public struct FlatCacheKey: Equatable, Hashable {
-
     let typeName: String
     let id: String
-
```

Comment:
As you might know already, Swift complier automatically synthesising the `Hashable` and `Equtable` Protocols. Even additionally avoiding unnecessary collision. (Matt also talks more about that [here](https://nshipster.com/hashable/))